### PR TITLE
News post for release-20180923.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
 - [x] Update changelog link to include any last minute changes
 - [x] Update mod sdk link
 - [x] Update release ID

screen shot 2018-09-22 at 16 29 09 2

Ping @IceReaper and @FiveAces to confirm that you are okay with being featured here.

### CODE REVIEW

**Comment by pchote** [comment]
Updated.

File: content/news/release-20180923.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+With two months of successful playtesting behind us, we are pleased to finally announce [Release 20180923](/download/). The new release brings players up to date with development from January to July 2018, which includes some interesting new features and welcome fixes despite a significant reduction in our development manpower.
+
+One of the themes in this release is integration: a new player account system bridges your in-game persona with the [OpenRA Forum](https://forum.openra.net); improvements to the game client and server infrastructure enable a more usable [online server list](/games/), and a new strategy for Linux packaging allows us provide baseline support for more players while reducing our maintenance burden.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180825-badges.png" width="600" alt="Player badges">
```
[the remark this answers is not in the document]
Comment (pchote) [reply content/news/release-20180923.md]: As discussed in IRC I resolved this by removing the games list screenshot.

**Comment by FiveAces** [comment]
@pchote Absolutely, sounds great! Thumbs up from my side!

File: content/news/release-20180923.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* Overhauled Chronoshift-return behaviour in Red Alert
+* Disabled Kill Bounties in the Red Alert campaign and default multiplayer configuration
+* Fixes for several crashes and gameplay bugs in release-20180307
+* A collection of community balance changes in [Red Alert](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1#ra-balance-changes) and [Tiberian Dawn](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1#cnc-balance-changes)
+
+As always, we suggest you read the [changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1) for more complete information, and head over to the [download](/download/) page to get started with the new release!
+
+<hr />
+
+The latest [Mod SDK](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20180825) releases contain major improvements over the 20180307 release:
```
Comment (reaperrr) [inline content/news/release-20180923.md]: I guess this should have been updated to 29180923?

Comment (abcdefg30) [reply content/news/release-20180923.md]: Good catch! #402

**Pull request merged**